src/main/java/frc/robot/commands/RelinquishCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Acquisition;
+
+public class RelinquishCommand extends Command {
+    private Acquisition acquisition = Acquisition.getInstance();
+    private Timer timer = new Timer();
+    private double timeToDrop = 1;
```

Comment:
Create a constant for this time constraint instead of defining it within the command.